c++/src/kj/compat/http.h

```diff
@@ -80,7 +80,8 @@ namespace kj {
   MACRO(NOTIFY) \
   MACRO(SUBSCRIBE) \
   MACRO(UNSUBSCRIBE) \
-  MACRO(QUERY)
+  MACRO(QUERY) \
+  MACRO(BAN)
```

Comment:
Looks like both QUERY and BAN have been mislabeled as being from "UPnP" (comment on next line).

QUERY should reference the RFC.

For BAN we should write:

> Non-standard method name requested by a Cloudflare customer.